### PR TITLE
documents identity management

### LINKED ISSUE #1199: Document multi-tenancy feature
/kind documentation

**Describe the solution you'd like**
Document how to utilize `VSphereClusterIdentity` and the new multi-tenancy feature.

### PR DESCRIPTION
**Pull request by gab-satchi**
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1199 

```release-note
NONE
```

### CODE REVIEW

File: docs/identity_management.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Identity Management
```
Comment (yastij) [inline docs/identity_management.md:1]: can you drop a link in the getting started guide to this ?

**Review by yastij — COMMENTED** [review]
/approve

File: docs/identity_management.md (hunk 1 above)
Comment (gab-satchi) [reply docs/identity_management.md:1]: yep added. hopefully that's the correct way to link it

**Pull request merged**